src/Footer/Footer.css

```
@@ -0,0 +1,11 @@
+.app > .footer {
+	background-color: bisque;
+	border-radius: var(--app-padding);
+	/* bottom: 0; */
+	/* margin: 0; */
+	margin-top: var(--app-padding);
+	padding: var(--app-half-padding) 0;
```

The same note I mentioned about the CSS vars in the Header PR applies here - I don't think this CSS will work w/o the CSS vars in this file; in addition, I'm not sure we want spacing (as Material UI would refer to it) to be defined in CSS rather than declared via attributes in the TSX file, on the MUI components

```suggestion
	border-radius: 5;
```